rules/expiring-todo-comments.js

```diff
@@ -207,10 +207,12 @@ function semverComparisonForOperator(operator) {
 const create = context => {
 	const options = {
 		terms: ['todo', 'fixme', 'xxx'],
+		ignore: [],
 		ignoreDatesOnPullRequests: true,
 		allowWarningComments: true,
 		...context.options[0]
 	};
+	const ignoreRegexes = options.ignore.map(ignore => new RegExp(ignore));
```

Comment:
Would be nice to allow regex literal here, people may want do `/ignore/i`

Reply:
check this https://github.com/sindresorhus/eslint-plugin-unicorn/pull/431

Reply:
Resolved.

Mimicking the linked PR, I added the `'u'` flag by default, and set array in schema to `uniqueItems: true`.

Reply:
Can you  also use  some explanation and  example from  that PR?

Reply:
Updated docs to include examples of both string and regex and an small explainer on escaping characters.